packages/clerk-js/src/ui/components/SignUp/SignUpContinue.tsx

```diff
@@ -175,6 +175,7 @@ function _SignUpContinue() {
                 <SignUpSocialButtons
                   enableOAuthProviders={showOauthProviders}
                   enableWeb3Providers={showWeb3Providers}
+                  continueSignUp
```

Comment:
https://github.com/clerk/javascript/blob/ff52e7905c0f492c2cd2eac8bb8ece3e726237b7/packages/clerk-js/src/core/resources/SignUp.ts#L207-L208

Since `continueSignUp` wasn't being provided, then a new `SignUp` object got created on the redirect.

Reply:
I might be missing something here and this could ended up introducing side effects with other flows, so please let me know 🙇🏻

Reply:
❓Shouldn't the absence of `continueSignUp` create issues for the rest of oath providers ? Did we verify that those were failing as well, because if they were successful that would seem weird.

Reply:
Good point, I haven't verified yet - what are other flows that someone could fall on `ContinueSignUp` so that I can verify that?

Reply:
We have verified (offline) that flows work as expected